cartography/intel/gcp/apigateway.py

```diff
@@ -147,9 +212,68 @@ def get_api_configs(apigateway: Resource, project_id: str) -> List[Dict]:
         else:
             raise
 
+@timeit
+def get_api_config_policy_users(apigateway: Resource, configs: List[Resource],project_id: str) -> List[Dict]:
+    """
+        Returns a list of users attached to IAM policy of an API Config within the given project.
+
+        :type apigateway: The GCP APIGateway resource object
+        :param apigateway: The APIGateway resource object created by googleapiclient.discovery.build()
+
+        :type api: List
+        :param api: The list of APIs
+
+        :type project_id: str
+        :param project_id: Current Google Project Id
+
+        : type locations: List
+        : param Location: List of locations
+
+        :rtype: list
+        :return: List of api config iam policy users
+    """
+    users = []
+    try:
+        for config in configs:
+            config['iam_policy'] = apigateway.projects().locations().apis().configs().getIamPolicy(\
+                resource=config.get('name')).execute()
+            bindings = config.get('iam_policy',{}).get('bindings',[])
+            for binding in bindings:
+                for member in binding['members']:
```

Comment:
same as above. We should do this for all types of iam entities (users, roles, service accounts, etc.). Also, is there a way we can do this in a generic manner?

Instead of writing this logic for each service (apigateway, kms, functions, etc). we have a generic class which does this association of iam entities and resource.